### PR TITLE
[ASUSGAB] Add Portuguese, Arabic translation to Pdf xblock 

### PR DESCRIPTION
**Pull request by AndreyLykhoman**
*Description:*  Add Portuguese, Arabic translation to Pdf xblock. 
*Usage* You must add trabslation from text.po this xblock to django.po in platform. And generate .mo file.
*Youtrack tasks:* 
[sga]
https://youtrack.raccoongang.com/issue/ASUSGAB-37
https://youtrack.raccoongang.com/issue/ASUSGAB-22 
[ytp]
https://youtrack.raccoongang.com/issue/ASUYT-93

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,30 +4,32 @@
...
+            for fname in files:
+                data.append(os.path.relpath(os.path.join(dirname, fname), pkg))
 
     return {pkg: data}
 
 
 setup(
     name='xblock-pdf',
-    version='v1.0.0-rg',
+    version='v1.1.1',
```
Comment (idegtiarov) [inline setup.py]: following rg versioning, please.

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: pdf/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from pdf import PdfBlock
```
Comment (idegtiarov) [inline pdf/__init__.py]: It seems to be a redundant change.

**Review by idegtiarov — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,6 @@
 *.pyc
 
 node_modules/
+venv/
```
Comment (idegtiarov) [inline .gitignore]: Local settings are not required to be added to the repository.

**Review by idegtiarov — CHANGES_REQUESTED** [review]

**Comment by AndreyLykhoman** [comment]
I deleted extra lines and changed setup version.

File: requirements.txt [hunk 4]
```diff
@@ -1 +1,4 @@
+git+https://github.com/edx/xblock-utils.git@v1.0.4#egg=xblock-utils==1.0.4
+git+https://github.com/edx/XBlock.git@xblock-0.5.0#egg=XBlock==0.5.0
 flake8==2.4.0
+-e .
```
Comment (idegtiarov) [inline requirements.txt]: not needed

File: requirements.txt [hunk 5]
```diff
@@ -1 +1,4 @@
+git+https://github.com/edx/xblock-utils.git@v1.0.4#egg=xblock-utils==1.0.4
```
Comment (idegtiarov) [inline requirements.txt]: Should be set on the platform level

File: requirements.txt [hunk 6]
```diff
@@ -1 +1,4 @@
...
+git+https://github.com/edx/XBlock.git@xblock-0.5.0#egg=XBlock==0.5.0
```
Comment (idegtiarov) [inline requirements.txt]: Please clarify why do we need this lib?

**Review by idegtiarov — CHANGES_REQUESTED** [review]

**Comment by AndreyLykhoman** [comment]
@idegtiarov 
I removed extra requirements. Please review.

**Review by idegtiarov — APPROVED** [review]

**Pull request merged**